Skimpy/src/BusinessLogic/DBConnect.java

```diff
@@ -67,6 +68,44 @@ public Food getFoodData(String ID){ //table is the name of the table within the
 			return null;
 		}
 	}
+	
+	//TODO: Roo, this is my DBConnect method. If you run DBConnect() it will run your method because of polymorphism.
+	//However, my method is based on the method I wrote *before* you made updates to DBConnect. Could you update this so it works like your new method?
+	
+	public DBConnect(String db) //db is the name of the database. 

```

Comment:
Because it lets me connect to a database other than the default, without
changing the method itself.
Actually don't worry about it, I can make the changes myself. Both methods
will work fine side by side, so you can run the default method without
passing a var.

F

On Fri, Feb 13, 2015 at 1:57 PM, Ruaraidh Macfarlane <
notifications@github.com> wrote:

> In Skimpy/src/BusinessLogic/DBConnect.java
> <https://github.com/fsocko/Skimpy/pull/11#discussion_r24664990>:
>
> > @@ -67,6 +68,44 @@ public Food getFoodData(String ID){ //table is the name of the table within the
> >  			return null;
> >  		}
> >  	}
> > +	
> > +	//TODO: Roo, this is my DBConnect method. If you run DBConnect() it will run your method because of polymorphism.
> > +	//However, my method is based on the method I wrote *before* you made updates to DBConnect. Could you update this so it works like your new method?
> > +	
> > +	public DBConnect(String db) //db is the name of the database.
>
> why do you need this parameter?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fsocko/Skimpy/pull/11/files#r24664990>.
>